### PR TITLE
chore(release) bump Chart.yaml version to 1.1.3

### PR DESCRIPTION
**Pull request by shalier**
**Description**:
Bumping the Chart.yaml version per the release guide

Signed-off-by: Shalier Xia <shalierxia@microsoft.com>

**Affected area**:
| Functional Area            |     |
| -------------------------- | --- |
| Other                      | [ x] |

Please answer the following questions with yes/no.

1. Does this change contain code from or inspired by another project?
    -   Did you notify the maintainers and provide attribution?

2. Is this a breaking change?

3. Has documentation corresponding to this change been updated in the [osm-docs](https://github.com/openservicemesh/osm-docs) repo (if applicable)?

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ## Release v1.1.0
 
 ### Notable changes
-
+- Certificate rotation now ignores monotonic clock readings when checking expiration
```
Comment (jaellio) [inline docs/release_notes.md:6]: Should we add a separate release note for patch releases?

Comment (shalier) [reply docs/release_notes.md:6]: Like a different section from notable changes or like a header under notable changes?
I added the previous patch release change under Notable changes though, so then maybe we'd want to implement this for releases going forward?

Comment (shashankram) [reply docs/release_notes.md:6]: You should have a separate note for each release, including patch releases. This file typically is used to populate the final release notes. Regardless, this should accurately convey the information for releases and not update the release notes of an already released version.

**Review by shashankram — CHANGES_REQUESTED** [review]
Thanks for the change. Could you create a separate release note section for the patch release?

File: docs/release_notes.md (hunk 1 above)
Comment (shalier) [reply docs/release_notes.md:6]: Added sections for v1.1.2 and v1.1.3 and included the notable changes for each

**Review by jaellio — APPROVED** [review]

**Review by keithmattix — APPROVED** [review]

**Pull request merged**